aws/elastic-beanstalk/elastic-beanstalk.tf

```diff
@@ -0,0 +1,594 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# VARIABLES / LOCALS / REMOTE STATE
+# ----------------------------------------------------------------------------------------------------------------------
+
+variable "solution_stack_name" {
+  default = "64bit Amazon Linux 2018.03 v2.12.14 running Docker 18.06.1-ce"
```

Comment:
Please add descriptions to all variables.

Reply:
Added descriptions